install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -2999,7 +2999,19 @@
 
 if (isFroxlorVersion('0.9.34-dev4')) {
 
-    showUpdateStep("Updating from 0.9.34-dev4 to 0.9.34 final");
+    showUpdateStep("Updating from 0.9.34-dev4 to 0.9.34-dev5", false);
```

Comment:
wrong place for DB update, people running >0.9.34-dev5 will never get these, please always add a new -dev version to the bottom of the update_0.9.inc.php file